packages/core/src/views/account-center/Maximized.svelte

```diff
@@ -197,24 +199,16 @@
   }
 </style>
 
+<svelte:window bind:innerWidth />
+
 {#if disconnectConfirmModal}
   <DisconnectAllConfirm
     onClose={() => (disconnectConfirmModal = false)}
     onConfirm={disconnectAllWallets}
   />
 {/if}
 
-<div
-  in:fly={{
-    delay: position.includes('top') ? 100 : 0,
-    duration: 600,
-    y: position.includes('top') ? 56 : -76,
-    easing: quartOut,
-    opacity: 0
-  }}
-  on:click|stopPropagation={hideWalletRowMenu}
-  class="outer-container"
->
+<div on:click|stopPropagation={hideWalletRowMenu} class="outer-container">
```

Comment:
Curious as to why the animation was removed here?

Reply:
Removing the animations was necessary for achieving the smooth transition required in the Figma design. Tweaking the animation didn't achieve the required effect. The configurations can always switch to the `Minimized.svelte` file if the micro view is not wanted.

Reply:
I will most likely return the animation back since the new minimal center will default to` false` for desktop.